modules/toast/react/lib/Toast.tsx

```diff
@@ -97,13 +97,20 @@ export default class Toast extends React.Component<ToastProps> {
         padding={PopupPadding.s}
         handleClose={onClose}
         closeIconSize={IconButtonSize.Small}
+        role={!onClose && !onActionClick ? 'status' : undefined}
+        aria-live={!onClose && !onActionClick ? 'polite' : undefined}
+        aria-atomic={!onClose && !onActionClick ? true : undefined}
```

Comment:
if the toast has no link or or focusable element, these are the attributes it needs for accessibility

Reply:
I'm curious about `undefined` instead of `false` here. I'm not sure this is incorrect, but I'm wondering if I'm missing some details.